src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -41,51 +48,75 @@ public ClaimService(
+                .getById(claimId)
+                .orElseThrow(() -> new NotFoundException("Claim not found by id " + claimId))
+        );
     }
 
     public List<Claim> getClaimBySubmitterId(String submitterId) {
-        return claimRepository.getBySubmitterId(submitterId);
+        return claimRepository.getBySubmitterId(submitterId).stream()
+            .map((claim) -> calculateAndPopulateTotalAmount(claim)).collect(Collectors.toList());
     }
 
     public Claim getClaimByLetterHolderId(String id) {
-        return claimRepository
-            .getByLetterHolderId(id)
-            .orElseThrow(() -> new NotFoundException("Claim not found for letter holder id " + id));
+        return calculateAndPopulateTotalAmount(
+            claimRepository
+                .getByLetterHolderId(id)
+                .orElseThrow(() -> new NotFoundException("Claim not found for letter holder id " + id))
+        );
     }
 
     public Claim getClaimByExternalId(String externalId) {
-        return claimRepository
-            .getClaimByExternalId(externalId)
-            .orElseThrow(() -> new NotFoundException("Claim not found by external id " + externalId));
+        return calculateAndPopulateTotalAmount(
+            claimRepository
+                .getClaimByExternalId(externalId)
+                .orElseThrow(() -> new NotFoundException("Claim not found by external id " + externalId))
+        );
     }
 
     public Optional<Claim> getClaimByReference(String reference, String authorisation) {
         String submitterId = userService.getUserDetails(authorisation).getId();
-        return claimRepository
-            .getByClaimReferenceAndSubmitter(reference, submitterId);
+
+        Optional<Claim> optionalClaim = claimRepository.getByClaimReferenceAndSubmitter(reference, submitterId);
+
+        if (optionalClaim.isPresent()) {
+            Claim claim = optionalClaim.get();
+            optionalClaim = Optional.of(calculateAndPopulateTotalAmount(claim));
+        }
+
+        return optionalClaim;
     }
 
     public Optional<Claim> getClaimByReference(String reference) {
-        return claimRepository
-            .getByClaimReferenceNumber(reference);
+
+        Optional<Claim> optionalClaim = claimRepository.getByClaimReferenceNumber(reference);
+
+        if (optionalClaim.isPresent()) {
+            Claim claim = optionalClaim.get();
+            optionalClaim = Optional.of(calculateAndPopulateTotalAmount(claim));
+        }
+
+        return optionalClaim;
     }
 
     public List<Claim> getClaimByExternalReference(String externalReference, String authorisation) {
         String submitterId = userService.getUserDetails(authorisation).getId();
-        return claimRepository.getByExternalReference(externalReference, submitterId);
+        return claimRepository.getByExternalReference(externalReference, submitterId)
+            .stream().map((claim) -> calculateAndPopulateTotalAmount(claim)).collect(Collectors.toList());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1612)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "claim" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1611)